src/gui/virtualstudio.cpp

```diff
@@ -431,6 +431,10 @@ void VirtualStudio::applySettings()
     m_previousBuffer     = m_bufferSize;
     m_previousInput      = m_inputDevice;
     m_previousOutput     = m_outputDevice;
+
+    // make sure device change signals are emitting
```

Comment:
Don't really need the comment here. These just weren't included before because all they do is make sure the interface shows the right devices. When the applySettings function could only be called from one place in the UI there was no need to update the UI since that was driving the change and already in sync.